src/components/Reports/Reports.js

```diff
@@ -50,6 +52,28 @@ const Reports = ({ dependencies: { datahubClient } }) => {
     }));
   };
 
+  useEffect(() => {
+    const fetchVisitsByPeriod = async () => {
+      const visitsWithoutEmail = await datahubClient.getTotalVisitsOfPeriod({
+        domainName: state.domainSelected.name,
+        dateFrom: state.dateFrom,
+        dateTo: state.dateTo,
+        emailFilter: 'without_email',
+      });
+      const visitsWithEmail = await datahubClient.getTotalVisitsOfPeriod({
+        domainName: state.domainSelected.name,
+        dateFrom: state.dateFrom,
+        dateTo: state.dateTo,
+        emailFilter: 'without_email',
```

Comment:
Shouldn't be `with_email` or something like that in this case?

Reply:
yes sorry you are right

Reply:
Shouldn't this data be in a constant?

Reply:
maybe we can, but i didn't put there because this component its temporal until the new one its done, but maybe we can add